src/core/shape/vertex.js

```diff
@@ -896,6 +899,56 @@ p5.prototype.quadraticVertex = function() {
  * @alt
  * 8 points making 4 lines
  *
+ * @example
+ * <div>
+ * <code>
+ * // Shape with a hollow section in the middle. Click to change the number of sides.
+ * // Custom shapes drawn with beginShape() would make use of the tessellator by default.
+ * // Custom shapes with different z-values (i.e non-coplanar shapes) would not use the tessellator.
+ * // So, it is recommended to draw all custom shapes on a flat z-plane.
```

Comment:
This might be specific to my screen but these comments are bit too long to view easily in my browser. I know the example is already a bit long vertically but you might have to make some of these statements multi-line or shorter.

> // Custom shapes drawn with beginShape() would make use of the tessellator by default.

I don't know how I feel about the introduction of the word tessellator in this example. It stands out a lot and someone without an understanding of what that means might be intimidated by it. 

Also this tense might be a bit strange. I think `will` might work better.

> // So, it is recommended to draw all custom shapes on a flat z-plane

I don't know that we should explicitly suggest that people don't make volumetric shapes.

As a general suggestion something more in the range of 
```
 * // Click to change the number of sides.
 * // In WebGL mode, custom shapes will only
 * // display hollow fill sections when
 * // all calls to vertex() use the same z-value.
```
might work better for this comment section. Feel free to change that up as you see fit but those are the general suggestions. I think targeting the least-expert user is usual a good call.